docs/contributors.rst

```diff
@@ -4,10 +4,10 @@
 Contributors
 ============
 
-All contributors to the libCellML software project agree to the terms and conditions of the APACHE 2 license.
+All contributors to the libCellML software project agree to the terms and conditions of the `Apache v2.0 <https://opensource.org/licenses/Apache-2.0>`_ license.
 
 List of Contributors
 ====================
 
- * University of Auckland
- * VPR
+ * `University of Auckland <https://www.auckland.ac.nz/>`_
+ * `VPR <http://www.virtualrat.org/>`_
```

Comment:
What exactly is a "contributor" in this context?

Reply:
Good point, not least since you guys in Oxford also contributed to the code. Happy to change this to whatever you feel like, and add Oxford to the list.

@nickerso: re VPR, should it still/really be listed?

Reply:
I believe the idea was to set up a contributor agreement and this list of contributors would list everyone who ever contributed to libCellML. I lieu of that, we initialised the list with "current" funding sources and haven't gone back. Would be good to update this to an actual list of contributors and an acknowledgement of funding that has been received at any point in the project life cycle. (yes, VPR should still be listed)

Reply:
but as a separate issue, no need to do that now.